web/client/components/geostory/layouts/sections/__tests__/Banner-test.jsx

```diff
@@ -0,0 +1,212 @@
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+
+        const backgroundContainer = container.querySelector('.ms-section-background-container');
+        expect(backgroundContainer).toExist();
+        expect(backgroundContainer.clientHeight).toBe(DEFAULT_BANNER_HEIGHT);
+    });
+    it('Banner rendering with Map as background', () => {
+        const VIEW_HEIGHT = 834;
+        const CONTENTS = [
+            {
+                id: '000',
+                type: "text",
+                background: {
+                    type: 'map'
+                }
+            }
+        ];
+        ReactDOM.render(
+            <Provider store={{
+                getState: () => {},
+                subscribe: () => {},
+                dispatch: () => {}
+            }}>
+                <Banner contents={CONTENTS} viewHeight={VIEW_HEIGHT} cover mode="edit"/>
+            </Provider>
+            , document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner');
+        expect(el).toExist();
+
+        const backgroundContainer = container.querySelector('.ms-section-background-container');
+        expect(backgroundContainer).toExist();
+        expect(backgroundContainer.clientHeight).toBe(VIEW_HEIGHT);
+        const contentToolbar = container.querySelector('.ms-content-toolbar');
+        expect(contentToolbar).toExist();
+        testToolbarButtons(["pencil", "height-auto", "map-edit", "size-extra-large", "align-center", "dropper", "trash"], container);
+    });
+    it('should apply expandable background class on media map', () => {
+        const VIEW_HEIGHT = 834;
+        const CONTENTS = [
+            {
+                id: '000',
+                type: "text",
+                background: {
+                    type: 'map'
+                }
+            }
+        ];
+        ReactDOM.render(
+            <Provider store={{
+                getState: () => {},
+                subscribe: () => {},
+                dispatch: () => {}
+            }}>
+                <Banner contents={CONTENTS} expandableMedia viewHeight={VIEW_HEIGHT} cover mode="edit"/>
+            </Provider>
+            , document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('.ms-section-banner.ms-expandable-background');
+        expect(el).toExist();
+    });
+    it('should render video toolbar on background (fit cover)', () => {
+        const CONTENTS = [{
+            id: '000',
+            type: 'column',
+            background: {
+                type: 'video',
+                fit: 'cover'
+            },
+            contents: [{
```

Comment:
Please remove `type: 'column'` and `contents:[...` because they are not used by banner